CameraOverlay/config.py

```diff
@@ -48,8 +48,8 @@ def read_configs(directory=CURRENT_DIRECTORY):
         with open(configs_file) as file:
             configs = json.load(file)
 
-    current_device = os.getenv('MHP_CAMERA')
-    configs['device'] = current_device
+    configs['device'] = os.getenv('MHP_CAMERA')
+    configs['bike'] = os.getenv('MHP_BIKE')
```

Comment:
Thoughts on having `MHP_BIKE` be set as an argument? We can set the default argparse value as an environment variable otherwise have it set to V2? https://stackoverflow.com/questions/10551117/setting-options-from-environment-variables-when-using-argparse

Reply:
What advantage would having it set by an arguement have? Actually, I was wondering if we should have it stored in `config.json` with the active overlay. That way, this stuff could eventually be able to be set from the dashboard, along with other settings (brightness/contrast, flip vertical axis, whatever else becomes important)

Reply:
I don't think we should alter the bike version via dashboard cos that should be set. 
However, it would be good as an argument because when we test it out, we can easily set it through the argument instead of changing the `.env` file.

Reply:
Let's say we added it as a command line argument - how would `switch_demo.py` know what bike to use in the argument?

Reply:
I would get the default value from an environment variable!

Reply:
Done. Had to add a lot of boilerplate code though. I'm thinking maybe we should restructure things so that overlays are started with `python overlay.py -o top_strip` or something, so we don't have to handle command line arguments for each separate script.

Reply:
I do like that. We can create an OverlayFactory! I suggest you to read some creational design patterns one day. Factory and Builder patterns might be useful here. https://refactoring.guru/design-patterns/creational-patterns